other/bootstrap_daemon/tox_bootstrap.service

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=Tox DHT Bootstrap Daemon
+After=network.target
+
+[Service]
+Type=forking
+PermissionsStartOnly=true
+ExecStartPre=/usr/bin/touch /var/run/tox-bootstrapd.pid
```

Comment:
systemd deletes PIDFile on "stop" signal. So I'm just creating new pidfile for tox-bootstrap, because it's user don't have write permission to /var/run.

Are you planning to stay in foreground, if PID file not specified?